projects/store-test/src/app/app.component.ts

```diff
@@ -16,5 +17,7 @@ export class AppComponent {
 		this.store.dispatch(actions.users.effects.set());
 		this.store.dispatch(actions.users.effects.add({ payload: 'serguey' }));
 		this.store.dispatch(actions.paging.effects.set());
+
+		this.store.select(selectors.admins.selectAll).pipe(tap(console.log)).subscribe();
```

Comment:
Console log

Reply:
Zoals aangegeven, dit is de test applicatie voor intern gebruik dus hier mag dag 👍